src/pages/FlowRun/Details-Tile.vue

```diff
@@ -146,7 +146,7 @@ export default {
                     ? 'Prefect Scheduler'
                     : flowRun.created_by
                     ? flowRun.created_by.username
-                    : 'A nonexistant user'
+                    : 'Ad hoc run'
```

Comment:
Open to better wording suggestions

Reply:
Random ideas:
- external process
- API call
- POST request
- just the word "user" (<-- this one best matches the alternate "Prefect Scheduler")

Reply:
Yeah I kind of like _user_/_User_ since it is literally created by _a_ user lol

Reply:
What about removing this field entirely for Server? Having "User" suggests that user as a concept exists in the system, which it doesn't.

Reply:
The reason I left it in is because I like that we show that the run was auto scheduled with the Prefect Scheduler and something different for if it was created manually. I don't feel too strongly about it and am fine for removing it though

Reply:
Yeah I think auto-scheduled information is helpful so definitely in favor of keeping that but hiding the user portion for Server